client/my-sites/media-library/list-plan-promo.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import identity from 'lodash/identity';
```

Comment:
Not sure if this is better or not, but for consistency's sake we have been importing directly from `lodash`

```js
import { identity } from 'lodash';
```